### PR TITLE
[T] (quests) quest-weak_enriching_water

### PR DESCRIPTION
**Pull request by AnzhiZhang**
Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/weak_enriching_water.snbt [hunk 1]
```diff
@@ -387,7 +387,7 @@
 		{
 			x: 0.0d
 			y: -6.0d
-			description: ["Raw mixed stone"]
+			description: ["粗混合石Raw mixed stone"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/weak_enriching_water.snbt]: 1

File: config/ftbquests/quests/chapters/weak_enriching_water.snbt [hunk 2]
```diff
@@ -399,7 +399,7 @@
 		{
 			x: 0.0d
 			y: -7.5d
-			description: ["Can be used to make spray cans which can be used to paint stone blocks"]
+			description: ["用于制作可喷涂石块的喷雾罐Can be used to make spray cans which can be used to paint stone blocks"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/weak_enriching_water.snbt]: 1

**Comment by AnzhiZhang** [comment]
等 Caveopolis 翻译出来

File: config/ftbquests/quests/chapters/weak_enriching_water.snbt (hunk 1 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/weak_enriching_water.snbt]:
```suggestion
			description: ["粗混合石"]
```

File: config/ftbquests/quests/chapters/weak_enriching_water.snbt (hunk 2 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/weak_enriching_water.snbt]:
```suggestion
			description: ["用于制作可喷涂石块的喷雾罐"]
```

**Review by Cactusstudent — APPROVED** [review]
我之前翻译的居然都没问题，我就直接合了

**Pull request merged**